public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -122,6 +122,9 @@ export class FolderPicker extends PureComponent<Props, State> {
       folder = options.find((option) => option.value === initialFolderId) || null;
     } else if (enableReset && initialTitle) {
       folder = resetFolder;
+    } else if (initialTitle && initialFolderId === -1) {
+      // @TODO temporary, we don't know the id for alerting rule folder in some cases
```

Comment:
nit: should we track this in some overall issue so it doesn't get lost?

Reply:
you're right, will do